src/sentry_plugins/twilio/client.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from sentry_plugins.client import ApiClient
+
+
+class TwilioApiClient(ApiClient):
+    plugin_name = "twilio"
+    allow_redirects = False
+    twilio_messages_endpoint = "https://api.twilio.com/2010-04-01/Accounts/{0}/Messages.json"
+
+    def __init__(self, account_sid=None, auth_token=None, sms_from=None, sms_to=None):
```

Comment:
@ceorourke Having a default value of `none` seems to imply many of these are optional parameters even though I don't think they are.